src/components/SortBy.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { PropsType } from "../containers/SortByContainer";
+import * as objAssign from "object-assign";
+import { Store } from "azsearchstore";
+import { defaultCss } from "../utils/css";
+
+export type State = {};
+
+class SortBy extends React.Component<PropsType, State> {
+  render() {
+    const { fields, defaultField, resultCount, onSortChange } = this.props;
+    let css = objAssign({}, defaultCss, this.props.css);
+
+    if (resultCount === -1) return <div></div>;
```

Comment:
resultCount may always be -1 if user sets count to false. Can we try to use state.result.lastUpdated instead?